File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/ProjectJSONBean.java
package org.javamaster.invocationlab.admin.model.erd;

import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

/**
 * @author yudong
 * @date 2023/2/12
 */
@NoArgsConstructor
@Data
public class ProjectJSONBean {
    private List<ModulesBean> modules;
    private ProfileBean profile;
    private DataTypeDomainsBean dataTypeDomains;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/ProjectJSONBean.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen

/**
 * @author yudong
 * @date 2023/2/12
 */


@AllOpen
class ProjectJSONBean {
    var modules: MutableList<ModulesBean>? = null
    var profile: ProfileBean? = null
    var dataTypeDomains: DataTypeDomainsBean? = null
}
